frontend/src/pages/MeetingRoom/MeetingRoom.tsx

```diff
@@ -80,6 +82,7 @@ const MeetingRoom = (): ReactElement => {
 
   return (
     <div data-testid="meetingRoom" className="meetingRoom bg-darkGray-100 w-screen">
+      {isMobile() && <MobileHeader />}
```

Comment:
Note: I'm going to use `isMobile()` instead of the small port view hook here because we do not want this to be displayed on the small tabs ™️ on desktop

Reply:
Do we not?

Reply:
I guess we actually do. gotta tell em that they're being recorded, too. I'll change to small view port